assets/src/edit-story/components/library/panes/shared.js

```diff
@@ -36,76 +31,58 @@ const Tabs = styled.ul`
   list-style: none;
 `;
 
-const Tab = styled.li`
-  width: 64px;
-  height: 100%;
-  color: ${({ theme }) => theme.colors.fg.v1};
+const Icon = styled.div`
+  width: 60%;
+  height: 80%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 `;
 
-const Icon = styled.a`
-  color: inherit;
+const Tab = styled.li.attrs(({ isActive }) => ({
```

Comment:
This should really be a button to denote to screen readers it is clickable.

Reply:
See example [here](https://github.com/google/web-stories-wp/blob/40cdc3e9f92b4569c91b229fec8bfe6e9f09c8e3/assets/src/edit-story/components/inspector/inspectorTabs.js#L85-L92) and follows this [example](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html).

Reply:
It has [the interactive role `tab` which denotes to screen readers, that it's clickable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role). As discussed [here](https://github.com/google/web-stories-wp/pull/361#discussion_r382576516), I think this is the best semantic solution for this widget.